test/integration/vistor_can_view_items_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class VistorCanViewItemsTest < ActionDispatch::IntegrationTest
+
+  def create_items(num)
+    num.times do
+    Item.create(title: "latte", description: 'forthy deliciousness', price: 4.00, image: "google.com")
+    end
+  end
+
+  test 'an unregistered user can view items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.